### ISSUE TITLE
BlackWidow v3 ripple not working, keeps requesting accessibility access even though already granted

### ISSUE THREAD

**Issue by NotJB1** [1kc/razer-macos#504]
version 0.4.6, macOS Monterey 12.0.1
tried downgrading, works on version 0.4.5

**Comment by overcurrent** [comment]
Hi, this happened to me on M1 when the build is not code signed. Somehow OS X won't allow iohook module do what it wants to do without this.

@1kc I can see there's been couple of added hw features after 0.4.6. Do you want to create 0.4.7 (update packages.json and I can fetch the changes to my fork) out of the current master and I can build signed package this time again. Then you can download the release packages from my fork.

**Comment by overcurrent** [comment]
@1kc There is now a pull request waiting from my fork. I fetched the latest master, versioned it as 0.4.7 and signed the build.
Pull Request is here and link to universal dmg also there: https://github.com/1kc/razer-macos/pull/506

**Comment by overcurrent** [comment]
@NotJB1 We just released 0.4.7. You can get it here: https://github.com/1kc/razer-macos/releases/tag/v0.4.7

That should solve this issue and get you to the latest version. If so, this issue could be closed.
